src/components/menu-right/SetNextPoint.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div>
+    <div class="mb-1 field">
+      <b-switch v-model="isSetNextPointMode" data-test="set-next-point--switch"
+        >Set Next Point Mode</b-switch
```

Comment:
I think this should have a `size="is-small"` so it's the same size as the other components of menu-right. Thoughts?
| small | normal |
|-|-|
|<img width="248" alt="image" src="https://user-images.githubusercontent.com/29964426/102706079-ae554d00-4243-11eb-9160-843cda527f46.png">  |<img width="253" alt="image" src="https://user-images.githubusercontent.com/29964426/102706086-bad9a580-4243-11eb-98ca-1a76bbe98cfe.png">|